frontend/cypress/integration/nodes-page.spec.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="Cypress" />
+
+context("Nodes", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/nodes/validators");
+  });
+
+  // https://on.cypress.io/interacting-with-elements
+
+  it("Check for node page", () => {
+    cy.url().should("include", "/nodes/validators");
+    cy.get("#validator-node").find(".node-selector").contains("Validating");
+    cy.get("#online-node").find(".node-selector").contains("Online-nodes");
+    cy.get("#proposal-node").find(".node-selector").contains("Proposal-nodes");
+  });
+
+  it("Check validators tab", () => {
+    cy.url().should("include", "/nodes/validators");
+    cy.get("#validator-node").should("have.class", "node-selected");
+    cy.wait(3000)
+      .get(".node-row .node-row-title")
+      .within(($el) => cy.get($el).should("exist", $el.text()));
+    cy.get(".node-row .node-row-text .row div").within(($el) =>
+      cy.get($el).should("exist", $el.text())
+    );
+    cy.get(".node-row .node-row-txid").should("exist");
+  });
+
+  it("Check online nodes tab", () => {
+    cy.get("#online-node").click();
+    cy.get("#online-node").should("have.class", "node-selected");
+    cy.wait(3000)
+      .get(".node-row .node-row-title")
+      .within(($el) => cy.get($el).should("exist", $el.text()));
+    cy.get(".node-row .node-row-txid").within(($el) =>
+      cy.get($el).should("exist", $el.text())
+    );
+  });
+
+  it("Check proposal nodes tab", () => {
+    cy.get("#proposal-node").click();
+    cy.get("#proposal-node").should("have.class", "node-selected");
+    cy.wait(3000)
+      .get(".node-row .node-row-title")
+      .within(($el) => cy.get($el).should("exist", $el.text()));
```

Comment:
Well, I will just remove this part of the test for now since there could be no nodes in the proposal list at all.

```suggestion
```